scripts/jenkins/log-parser/openstack-mkcloud-rules.txt

```diff
@@ -18,12 +18,28 @@ ok /Warning: Turning on '--gpg-auto-import-keys'/
 ok /Warning: Permanently added .* to the list of known hosts/
 ok /WARNING: Device for PV .* not found or rejected by a filter/
 ok /WARNING: \w+ signature detected on .* offset \d+. Wipe it?/
-ok /perl-Error[ -]/
 ok /grep -v failed\b/
 
+# rpms containing "Error"
+ok /perl-Error[ -]/
+ok /libsamba-errors/
+
+# Crowbar upgrade messages
+ok /^\| Check ID +\| Passed \| Required \| Errors/
```

Comment:
It's just regular expression syntax.  That's output from `crowbarctl upgrade prechecks`.